tests/nwis_test.py

```diff
@@ -91,73 +92,78 @@ def test_get_record_site_value_types(site_input_type_list):
 # these specific queries represent some edge-cases and the tests to address
 # incomplete date-time information
 
+
 def test_inc_date_01():
     """Test based on GitHub Issue #47 - lack of timestamp for measurement."""
     site = "403451073585601"
     # make call expecting a warning to be thrown due to incomplete dates
     with pytest.warns(UserWarning):
-        df = get_record(site, "1980-01-01", "1990-01-01", service='gwlevels')
+        df = get_record(site, "1980-01-01", "1990-01-01", service="gwlevels")
     # assert that there are indeed incomplete dates
     assert any(pd.isna(df.index) == True)
     # assert that the datetime index is there
-    assert df.index.name == 'datetime'
+    assert df.index.name == "datetime"
     # make call without defining a datetime index and check that it isn't there
-    df2 = get_record(site, "1980-01-01", "1990-01-01", service='gwlevels',
-                     datetime_index=False)
+    df2 = get_record(
+        site, "1980-01-01", "1990-01-01", service="gwlevels", datetime_index=False
```

Comment:
My guess is the indent convention here is one tab right of the LHS? I guess it's the same as for loops and ifs and such.